lessons/basics/basics.md

```diff
@@ -120,10 +120,10 @@ iex> "dziękuję"
 Strings support line breaks and escape sequences:
 
 ```elixir
-iex(9)> "foo
```

Comment:
Not sure how I missed these, I usually configure `iex` to leave them off.

Thank you! :+1: